tests/integration/models/applicationInvite.spec.ts

```diff
@@ -0,0 +1,130 @@
+// tslint:disable-next-line:import-blacklist
+import * as m from 'mochainon';
+import {
+	balena,
+	BalenaSdk,
+	givenAnApplication,
+	givenLoggedInUser,
+} from '../setup';
+const { expect } = m.chai;
+
+const TEST_EMAIL = 'user.test@example.org';
+const TEST_MESSAGE = 'Hey!, Join my app on balenaCloud';
+const TEST_ROLE = 'developer';
+
+describe('Application Invite Model', function() {
+	givenLoggedInUser(beforeEach);
+	givenAnApplication(beforeEach);
```

Comment:
Since doing this in a beforeEach is a slow operation for the API, which ends up slowing the SDK tests a lot, please consider removing this from here and adding a `givenAnApplication(before);` inside the
`describe('given no application invite'` & `describe('given a single application invite'`.